auth-web/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth-web",
-  "version": "1.1.16",
+  "version": "1.1.17",
```

Comment:
Lewis, please bump this one up to 1.1.18

Reply:
Sure. I'll bump up this version in my next fix.

Reply:
FIY, the version is updated.